packages/account-v2/src/containers/TradingAssessmentForm/TestWarningModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv-com/ui';
+import { ACCOUNT_MODAL_REF } from '../../constants';
+
+type TTestWarningModal = {
+    handleSubmit: () => void;
+    isModalOpen: boolean;
+};
+
+export const TestWarningModal = ({ handleSubmit, isModalOpen }: TTestWarningModal) => {
```

Comment:
Maybe we need to move these modals to modules folder since tradershubt eam needs them in real aacount signup flow. Please help to check with tradersHub Team

Reply:
Checked with Traders Hub team, we will pass them the strings from the modal into a config file and send to them. They will create the Modals from their side

cc: @thisyahlen-deriv

Reply:
i think its better we get it raw like this so we have the leeway for future enhancements